package.json

```diff
@@ -82,6 +82,15 @@
     "patch-package": "patch-package",
     "build-xs-worker": "cd packages/xs-vat-worker && yarn build:xs-lin"
   },
+  "ava": {
+    "files": [
+      "packages/*/test/**/test-*.js"
```

Comment:
Does this interfere with / override `ava` settings in individual package's `yarn test`?

It looks like this is meant for use by a top-level `yarn test` which runs tests in all packages in parallel. When I last looked into doing that, I got stuck on the different `require` options we needed for different sets of tests (in particular importing `install-ses`, or `install-metering-and-ses`). Did you find a solution for that?

Reply:
> Does this interfere with / override `ava` settings in individual package's `yarn test`?

Only when run from the top level.  If `yarn test` is run in the specific package, the package's settings will take effect.

> It looks like this is meant for use by a top-level `yarn test` which runs tests in all packages in parallel. When I last looked into doing that, I got stuck on the different `require` options we needed for different sets of tests (in particular importing `install-ses`, or `install-metering-and-ses`). Did you find a solution for that?

Yes.  It turns out we only use `"require"` for two modules: `"esm"` (which appears in basically every `"ava"` config), and an enzyme setup script (only used in `packages/ui-components/package.json`).  So, the situation is not nearly as grim.

Reply:
Ok, I think @dtribble was once keen on using the AVA config to jam a `-r @agoric/install-ses` into e.g. all zoe tests, to avoid adding a line to the top of every test file, and that was going to push us to e.g. move all `@agoric/install-metering-and-ses` tests into one directory, and have two top-level AVA calls (with one set of `-r` options each). I think we talked him down from that :).

I've got `nodeArguments: [ "--expose-gc" ]` in the SwingSet package.json, to enable GC controls while testing. We may need to move that to the top-level package.json to make sure swingset tests work both from the top level and from `packages/SwingSet` (and also when e.g. Zoe does swingset-based tests, although if they're short enough, we can probably tolerate the inability to provoke GC).